Common/src/main/java/cn/stevei5mc/serverhelper/common/utils/CommonUtils.java

```diff
@@ -1,7 +1,23 @@
 package cn.stevei5mc.serverhelper.common.utils;
 
 public class CommonUtils {
-    public static String[] toArray(String input) {
-        return new String[]{input};
+
+    /**
+     * String ==> String[]
+     * @param string Input string
+     * @return String[]
+     */
+    public static String[] toArray(String string) {
+        return new String[]{string};
+    }
+
+    /**
+     * String ==> String[]
+     * Split lines ( "\n" split )
+     * @param string Input string
+     * @return String[]
+     */
+    public static String[] splitLines(String string) {
+        return string.split("\n");
```

Comment:
在 `Common/src/main/java/cn/stevei5mc/serverhelper/common/utils/CommonUtils.java` 中添加了 `splitLines` 方法。建议添加对输入参数的空值检查，以增强方法的健壮性。

```suggestion
/**
 * String ==> String[]
 * Split lines ( "\n" split )
 * @param string Input string
 * @return String[]
 */
public static String[] splitLines(String string) {
    if (string == null) {
        return new String[0];
    }
    return string.split("\n");
}

Reply:
这个无需